mne/io/boxy/boxy.py

```diff
@@ -0,0 +1,301 @@
+# Authors: Kyle Mathewson, Jonathan Kuziek <kuziek@ualberta.ca>
+#
+# License: BSD (3-clause)
+
+import re as re
+
+import numpy as np
+
+from ..base import BaseRaw
+from ..meas_info import create_info
+from ...utils import logger, verbose, fill_doc
+from ...annotations import Annotations
+
+
+@fill_doc
+def read_raw_boxy(fname, datatype='AC', preload=False, verbose=None):
+    """Reader for a BOXY optical imaging recording.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the BOXY data file.
+    datatype : str
+        Type of data to return (AC, DC, or Ph).
```

Comment:
I think defining AC and DC would make this more confusing (alternating current and direct current), these are short hands used by the vendor in the data file to indicate the background vs the modulated components of the detected light, DC offset vs AC signal

Reply:
If its confusing and needs explanation here then its sure to come up again in the future, so its probably worth defining what is actually being returned. It could either be here or in the brief tutorial section in this PR.

Reply:
okay, I can add this to the tutorial section